lib/request-wrapper.ts

```diff
@@ -132,45 +132,25 @@ export class RequestWrapper {
       this.axiosInstance.interceptors.request.use(
         (config) => {
           logger.debug('Request:');
-          try {
```

Comment:
I simply removed the try/catch blocks since logger.debug would not throw an exception.